ProcessMaker/Http/Controllers/ProcessController.php

```diff
@@ -36,7 +36,7 @@ public function index(Request $request)
         $status = $request->input('status');
         $processes = Process::all(); //what will be in the database = Model
         $processCategories = ProcessCategory::where('status', 'ACTIVE')->get();
-        $processCategoryArray = ['' => 'None'];
```

Comment:
We want this. It makes it so there is a default of none that validates that they must choose a category

Reply:
Ok I put it back. I originally took it out because it is being overwritten by `[null => '']` on line 83 in the blade template.